phenopy/__init__.py

```diff
@@ -1,2 +1,42 @@
 __project__ = 'phenopy'
 __version__ = '0.2.1'
```

Comment:
I think we should go for `0.3.0`. Only important when we do the release, but maybe we should do it in this PR so we don't forget.